components/raftstore/src/store/peer.rs

```diff
@@ -4536,6 +4539,18 @@ where
         });
         !has_overlapped_reads && !has_overlapped_writes
     }
+
+    pub fn adjust_raft_max_inflight_msgs_if_cfg_changed<T>(
+        &mut self,
+        ctx: &PollContext<EK, ER, T>,
+    ) {
+        if ctx.cfg.raft_max_inflight_msgs != self.raft_max_inflight_msgs {
+            self.raft_group
+                .raft
+                .adjust_max_inflight_msgs(self.peer_id(), ctx.cfg.raft_max_inflight_msgs);
```

Comment:
The `raft_max_inflight_msgs` is actually used by the leader for all other peers, so we should call `adjust_max_inflight_msgs` for all peers in the peer list and also should make sure incoming peers will set the same `max_inflight_msgs ` value

Reply:
Fixed, thanks!